crypto/secp256k1/secp256k1_nocgo_test.go

```diff
@@ -6,40 +7,8 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
-	"github.com/tendermint/tendermint/crypto/secp256k1"
 )
 
-// Ensure that signature verification works, and that
-// non-canonical signatures fail.
-// // Note: run with CGO_ENABLED=0 or go test -tags !cgo.
-func TestSignatureVerificationAndRejectUpperS(t *testing.T) {
```

Comment:
Seems innocuous but curious why we remove this test. Is it only concerning `nocgo` implementations ?